Audio_Steganography/TestingCode.py

```diff
@@ -0,0 +1,26 @@
+from Algos.Encrypt import Encrypt
+from Algos.Decrypt import Decrypt
+
+"""
+   [NOTE] Here we demostrate an use of the Encrypt and Decrypt algorithms 
+   We also play the audio file as well.
+"""
+# You can try with Forest.wav as well
+message_path = input("Enter path of Audio file")
+secret_path = input("Enter path of Secret message file")
```

Comment:
take input for message from input() function only, no file required for this

Reply:
Encrypt takes in path parameter. Taking input from user might have to change that. Also in case of large secret message file is beneficial. Should I make changes ?

Reply:
Yes make this change